python_apps/libretime_watch/libretime_watch/start_watching.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+import ConfigParser
+import pika, os, logging
+import json
+import psycopg2
+
+# initialize logging
+logfile= "/var/log/airtime/libretime_watch_cron.log"
+logging.basicConfig(format='%(asctime)s %(message)s',filename=logfile,level=logging.INFO)
+
+EXCHANGE="airtime-media-monitor"
+EXCHANGE_TYPE = "direct"
+ROUTING_KEY="filesystem"
+QUEUE="media-monitor"
+
+CONFIGFILE="/etc/airtime/airtime.conf"
```

Comment:
Can't this be imported from `readconfig`? That way you don't have to re-read all the config from the disk. You would just need to add the RabbitMQ stuff to `readconfig`